kfdef/kfctl_aws.yaml

```diff
@@ -374,9 +419,9 @@ spec:
     spec:
       auth:
         basicAuth:
-          password:
-            name: password
-          username: admin
+          password: $2y$12$ruoM7FqXrpVgaol44eRZW.4HWS8SAvg6KYVVSCIwKQPBmTpCm.EeO
```

Comment:
Does user need to pass the encoded string?

Reply:
Right, fixed.